maria-bucharest.md

```diff
@@ -0,0 +1,17 @@
+# Bucharest
+
+### Places to eat
+ - Jourmey Pub
+ - Van Gogh
+ - Eden
```

Comment:
We have a lot of old buildings and a Village Museum, where you can see old houses from the countryside. Also you can visit the Peasant Museum